OneCommunityGlobal/HighestGoodNetworkApp
# Nahiyan_Fixed Header UI

# Description
Fixed header UI by centering timer and hiding elements based on screen width

## Related PRS (if any):
This frontend PR is related to the dev backend.

## Main changes explained:
- Centered timer
- Font size decrease for timer
- Hiding clock icon when under 530px width
- Hiding owner message when between 1400px and 1700px

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to dashboard
6. verify that header UI is working properly from all viewports (300px and up)
7. Make sure there are no overlaps or items going out of view

## Screenshots or videos of changes:
Before: 
<img width="228" alt="Screenshot 2024-04-28 at 12 00 47 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71025942/a9df73a7-9c8d-4838-b2b7-b23e71075a84">
<img width="768" alt="Screenshot 2024-04-28 at 12 01 38 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71025942/f3a197b3-2efd-4121-b5ed-1bb5d854ab14">
<img width="234" alt="Screenshot 2024-04-28 at 12 02 22 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71025942/b7964131-d935-44bf-a9b1-f953d9e92fda">
